### PR TITLE
Add scan comment input

### PR DESCRIPTION
**Pull request by jhonkan**

### CODE REVIEW

**Review by vmuthupandian — APPROVED** [review]

**Review by aairey — APPROVED** [review]

File: mend-scanner/action.yml [hunk 1]
```diff
@@ -13,6 +13,9 @@ inputs:
   configFile:
     description: "Filename of Mend configuration if being used (including file path)"
     required: false
+  scan-comment:
+    description: Add an optional scan comment. By default, the action run URL is added.
```
Comment (aairey) [inline mend-scanner/action.yml:17]: @jhonkan I don't see this default being set, though?

File: mend-scanner/wss.sh [hunk 2]
```diff
@@ -107,8 +106,8 @@ set +e
 # Execute Unified Agent (2 settings)
 if [ -z  "$WS_CONFIG_FILE" ]; then
   java -jar wss-unified-agent.jar -noConfig true -apiKey "$WS_API_KEY" -project "$PROJECT_NAME_STR" "$PRODUCT_NAME_STR"\
-    -d . -wss.url "$WS_WSS_URL" -resolveAllDependencies true
+    -d . -wss.url "$WS_WSS_URL" -resolveAllDependencies true -scanComment "$SCAN_COMMENT"
 else
-  java -jar wss-unified-agent.jar -apiKey "$WS_API_KEY" -c "$WS_CONFIG_FILE" -d .
+  java -jar wss-unified-agent.jar -apiKey "$WS_API_KEY" -c "$WS_CONFIG_FILE" -d . -scanComment "$SCAN_COMMENT"
```
Comment (vmuthupandian) [inline mend-scanner/wss.sh:111]: perhaps should the -scanComment be added before -d . ?

Comment (vmuthupandian) [reply mend-scanner/wss.sh:111]: It is not picking up the input scan comment am passing.

**Comment by coliff** [comment]
Looks good - thanks! I've not tested this change but it has two approvals so we'll get this merged. 👍

**Comment by vmuthupandian** [comment]
alright thanks

File: mend-scanner/action.yml (hunk 1 above)
Comment (aairey) [reply mend-scanner/action.yml]: Seems there we have a default value now in `wss.sh` :) Could we add it here also?

Comment (msankaran0712) [reply mend-scanner/action.yml]: > Seems there we have a default value now in `wss.sh` :) Could we add it here also?

hey @aairey @jhonkan can someone create a PR for this please?

**Pull request merged**